lib/rules/declaration-property-value-no-unknown/index.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const { isPlainObject } = require('is-plain-object');
+const { fork, parse } = require('css-tree');
```

Comment:
I'm afraid this might well increase the package size, not agains this rule, just intresting how much. Also I think grammar maching can be bundled (and tree shaken) to solve this problem